linchpin/hooks/built_ins/azure_boot_log/boot_log.yml

```diff
@@ -12,11 +12,15 @@
       vault_pass: "{{ vault_pass }}"
     register: auth_var_out
     delegate_to: localhost
+    no_log: "{{ not auth_debug }}"
 
   - name: "set auth_var"
     set_fact:
       auth_var: "{{ auth_var_out['output'][credentials_profile] }}"
 
+  - debug:
```

Comment:
Is the debug necessary?

Also, one of the linting errors you'll see a lot of in ansible-lint is that all tasks must be named.  So if this is necessary, please name it

Reply:
no, I was trying to figure out why CI was failing.